pkg/server/test/write_assertions.go

```diff
@@ -10,22 +10,61 @@ import (
 	serverErrors "github.com/openfga/openfga/pkg/server/errors"
 	"github.com/openfga/openfga/pkg/storage"
 	"github.com/openfga/openfga/pkg/testutils"
+	"github.com/openfga/openfga/pkg/typesystem"
 	"github.com/stretchr/testify/require"
 	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
 )
 
 func TestWriteAssertions(t *testing.T, datastore storage.OpenFGADatastore) {
 	type writeAssertionsTestSettings struct {
-		_name   string
-		request *openfgapb.WriteAssertionsRequest
-		err     error
+		_name         string
+		request       *openfgapb.WriteAssertionsRequest
+		allowSchema10 bool
+		assertModel10 bool
+		err           error
 	}
 
 	store := testutils.CreateRandomString(10)
 
 	githubModelReq := &openfgapb.WriteAuthorizationModelRequest{
 		StoreId: store,
 		TypeDefinitions: []*openfgapb.TypeDefinition{
+			{
+				Type: "user",
+			},
+			{
+				Type: "repo",
+				Relations: map[string]*openfgapb.Userset{
+					"reader": {Userset: &openfgapb.Userset_This{}},
+					"can_read": {
+						Userset: &openfgapb.Userset_ComputedUserset{
```

Comment:
`typesystem.ComputedUserset("reader")` for short

This comment applies elsewhere where we use `&openfgapb.Userset_ComputedUserset`